api/v1alpha1/ionoscloudmachine_types.go

```diff
@@ -177,8 +177,8 @@ type IonosCloudMachineSpec struct {
 	// +immutable
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="lanID is immutable"
 	// +kubebuilder:validation:MinLength=1
-	//nolint:revive
-	LanID *string `json:"lanID,omitempty"`
+
+	NetworkID *string `json:"networkID,omitempty"`
```

Comment:
The empty line leads to a missing description in the rendered CRD.

Reply:
sigh